addons/hr_holidays/report/hr_holidays_reports.xml

```diff
@@ -1,28 +1,13 @@
 <?xml version="1.0"?>
 <odoo>
 
-        <record id="action_report_holidayssummary" model="ir.actions.report">
-            <field name="name">Time Off Summary</field>
-            <field name="model">hr.holidays.summary.dept</field>
-            <field name="report_type">qweb-pdf</field>
-            <field name="report_name">hr_holidays.report_holidayssummary</field>
-            <field name="report_file">hr_holidays.report_holidayssummary</field>
-        </record>
-
-        <record id="action_report_holidayssummary" model="ir.actions.report">
-            <field name="paperformat_id" ref="hr_holidays.paperformat_hrsummary"/>
-        </record>
-
-        <record id="action_report_holidayssummary2" model="ir.actions.report">
-            <field name="name">Time Off Summary</field>
-            <field name="model">hr.leave.allocation</field>
-            <field name="report_type">qweb-pdf</field>
-            <field name="report_name">hr_holidays.report_holidayssummary</field>
-            <field name="report_file">hr_holidays.report_holidayssummary</field>
-        </record>
-
-        <record id="action_report_holidayssummary" model="ir.actions.report">
-            <field name="paperformat_id" ref="hr_holidays.paperformat_hrsummary"/>
-        </record>
+    <record id="action_report_holidayssummary" model="ir.actions.report">
```

Comment:
I'd keep the original xmlid:

```suggestion
    <record id="action_report_holidayssummary2" model="ir.actions.report">
```

Reply:
hi @kba-odoo ,
the id referred when printing the report from python is the one in the pr:  https://github.com/niyasraphy/odoo/blob/16.0/addons/hr_holidays/wizard/hr_holidays_summary_employees.py#:~:text=return%20self.env.ref(%27hr_holidays.action_report_holidayssummary%27).report_action(employees%2C%20data%3Ddatas)

is it okay to keep or should i change here also.
thanks

Reply:
hi @kba-odoo ,
is it okay to keep or should i change in both place.

thanks in advance

Reply:
hi @kba-odoo ,
is it okay to keep or should i change in both place.

thanks in advance

Reply:
Just keep the xmlid then, to minimize the number of changes done :-)

thanks!